schema/v0/flag_manifest.json

```diff
@@ -36,87 +40,96 @@
           "$ref": "#/$defs/objectType"
         }
       ],
-      "required": ["flagType", "defaultValue"]
+      "required": [
+        "flagType",
+        "codeDefault"
+      ]
     },
     "booleanType": {
       "type": "object",
       "properties": {
         "flagType": {
           "type": "string",
-          "enum": ["boolean"]
+          "enum": [
+            "boolean"
+          ]
         },
-        "defaultValue": {
+        "codeDefault": {
```

Comment:
Switched from `defaultValue` to `codeDefault`. Please let me know if you prefer another term.